### PR TITLE
Introduce rule requiring Null Coalesce Equal Operator

### PR DESCRIPTION
**Pull request by carusogabriel**
This is a new rule that will only affect PHP 7.4+ codebases :)

### CODE REVIEW

File: lib/Doctrine/ruleset.xml [hunk 1]
```diff
@@ -250,6 +250,8 @@
     </rule>
     <!-- Require language constructs without parentheses -->
     <rule ref="SlevomatCodingStandard.ControlStructures.LanguageConstructWithParentheses"/>
+    <!-- Require usage of null coalesce operator equal operator (`??`) when possible -->
```
Comment (kukulich) [inline lib/Doctrine/ruleset.xml:253]: @carusogabriel it should be ??=

**Comment by kukulich** [comment]
@morozov because ??= exists only in PHP 7.4+

The PHPStan disallows ?:, this sniff requieres ??=

File: lib/Doctrine/ruleset.xml (hunk 1 above)
Comment (carusogabriel) [reply lib/Doctrine/ruleset.xml]: I've removed it, just leaving the name, same as the one bellow.

**Comment by greg0ire** [comment]
Thanks @carusogabriel !

**Pull request merged**